mlrun/projects/project.py

```diff
@@ -1211,8 +1211,22 @@ def set_workflow(
         :param ttl:           pipeline ttl in secs (after that the pods will be removed)
         :param args:          argument values (key=value, ..)
         """
-        if not workflow_path:
-            raise ValueError("valid workflow_path must be specified")
+
+        abs_workflow_path = (
+            os.path.join(self.spec.get_code_path(), workflow_path.removeprefix("./"))
+            if workflow_path.startswith("./")
+            else workflow_path
+        )
```

Comment:
does it mean that providing `workflow.py` without any `./` won't work? does it break backwards compatibility? do we want to support it?

Reply:
fixed and it will work now